src/videojs-hls.js

```diff
@@ -711,14 +709,24 @@ videojs.HlsHandler.prototype.selectPlaylist = function () {
       break;
     } else if (variant.attributes.RESOLUTION.width < width &&
```

Comment:
I think we're kind of inconsistent with this `if`. We guard the second clause with an `else if` even though we short-circuit the loop if the first clause is hit. Then in the third clause, we *don't* guard with an `else`, even though the second clause now short-circuits if it's run.